.github/workflows/deploy_prod.yml

```diff
@@ -33,6 +33,7 @@ jobs:
           EXPRESS_SDK_URL: https://dev.cc-embed.adobe.com/sdk/1p/v4/4.1.14/CCEverywhere.js
           SPLUNK_HEC__HEC_TOKEN: ${{ secrets.SPLUNK_HEC__HEC_TOKEN }}
           AIO_LOG_LEVEL: info
+          IS_CICD: 1
```

Comment:
You can use the GITHUB_ACTIONS variable to differentiate when the script is being run locally or by GitHub Actions.